internal/zinc-scripted/src/main/contraband-scala/sbt/internal/inctest/Project.scala

```diff
@@ -17,7 +17,7 @@ final class Project private (
     case _ => false
   }
   override def hashCode: Int = {
-    37 * (37 * (37 * (37 * (17 + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
+    37 * (37 * (37 * (37 * (37 * (17 + "Project".##) + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
```

Comment:
I would be happy to improve Contraband later, but we should merge this first since all other Zinc PR's are blocked by various breakages from io, lm etc.